app/components/geoblacklight/header_component.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Geoblacklight
+  class HeaderComponent < Blacklight::HeaderComponent
+    def not_landing_page?
+      controller_name == "catalog" && (helpers.has_search_parameters? || params[:action] == "show")
```

Comment:
I know this conditional already existed, so no need to change in this PR, but in general testing on the `controller_name == 'catalog'` is suboptimal because it's okay to give the controller powered by blacklight a different name.  It should not be required to be called `CatalogController`